src/Microsoft.Extensions.Logging.EventSource/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "version": "1.0.0-*",
+  "description": "A logger provider that sends data logged through ILogger interfaces to EventSource/EventListener infrastructure",
+  "packOptions": {
+    "tags": [
+      "EventSource",
+      "logging",
+      "ETW"
+    ],
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/aspnet/logging"
+    }
+  },
+  "buildOptions": {
+      "warningsAsErrors": true,
+      "keyFile": "../../tools/Key.snk",
+      "xmlDoc": true
+  },
+  "dependencies": {
+    "Microsoft.Extensions.Logging.Abstractions": "1.0.0-*",
+    "System.Diagnostics.Tracing": "4.1.0-*",
+    "System.Threading": "4.0.11-*",
+    "Newtonsoft.Json": "8.0.3"
+  },
+  "frameworks": {
+    "net46": {
+      "dependencies": {
+        "System.Runtime": "4.0.0"
+      },
+      "imports": [ "dotnet" ]
+    },
+    "netcoreapp1.0": {
```

Comment:
The reason why I went with netcoreapp is that when I am trying to consume Newtonsoft JSON from a project that targets netstandard, I am getting the following error:

```
Errors in C:\code\aspnet-logging-rc\src\Microsoft.Extensions.Logging.EventSource\project.json
    Package Newtonsoft.Json 8.0.3 is not compatible with netstandard1.3 (.NETStandard,Version=v1.3). Package Newtonsoft.Json 8.0.3 supports:
      - net20 (.NETFramework,Version=v2.0)
      - net35 (.NETFramework,Version=v3.5)
      - net40 (.NETFramework,Version=v4.0)
      - net45 (.NETFramework,Version=v4.5)
      - portable-dnxcore50+net45+win8+wp8+wpa81 (.NETPortable,Version=v0.0,Profile=net45+wp80+win8+wpa81+dnxcore50)
      - portable-net40+sl5+win8+wp8+wpa81 (.NETPortable,Version=v0.0,Profile=Profile328)
    One or more packages are incompatible with .NETStandard,Version=v1.3.
```

Now, I use Newtonsoft to serialize log data into JSON format. The serialization is quite simple, so I could do it manually and remove the dependency if overcoming the problem above is hard, but I would prefer to leverage Newtonsoft if possible